### DISCUSSION TITLE
NoRecipients

### DISCUSSION THREAD

**Discussion by nilsoncorrales** [postalserver/postal#1904] [Development discussions]
I get the following error every time I do it from postam

{
    "status": "error",
    "time": 0.0,
    "flags": {},
    "data": {
        "code": "NoRecipients",
        "message": "There are no recipients defined to receive this message"
    }
}

![image](https://user-images.githubusercontent.com/42094552/158254500-9b486c96-334d-4020-92a7-dac1b398fdb1.png)

**Comment by willpower232** [comment]
I believe the API expects a JSON body, not form-data
